micro-ui/web/micro-ui-internals/packages/modules/pgr/src/pages/citizen/Rating/SelectRating.js

```
@@ -11,7 +11,7 @@ const SelectRating = ({ parentRoute }) => {
   const dispatch = useDispatch();
   const history = useHistory();
 
-  let tenantId = Digit.SessionStorage.get("CITIZEN.COMMON.HOME.CITY")?.code || Digit.ULBService.getCurrentTenantId();
+  let tenantId = Digit.Utils.getMultiRootTenant()? Digit.ULBService.getStateId() : Digit.SessionStorage.get("CITIZEN.COMMON.HOME.CITY")?.code || Digit.ULBService.getCurrentTenantId();
```

Biome

[error] This let declares a variable that is only assigned once.

'tenantId' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)